GPU/GPUCommon.cpp

```diff
@@ -1013,6 +1000,7 @@ void GPUCommon::DoExecuteCall(u32 target) {
 
 	if (currentList->stackptr == ARRAY_SIZE(currentList->stack)) {
 		ERROR_LOG(G3D, "CALL: Stack full!");
+		// TODO: UpdateState(GPUSTATE_ERROR) ?
```

Comment:
I think I tested these cases and the PSP GE just ignores a RET with an empty stack or a CALL with a full one.  Been a bit, though.

-[Unknown]